tests/unit/plugins/service/test_event_consumer.py

```diff
@@ -104,6 +104,22 @@ def test_event_msg_append_to_history(mocker, pubsub_msg, raw_msg_data):
     assert expected == msg.history_log
 
 
+def test_event_msg_update_phase(mocker, pubsub_msg, raw_msg_data):
+    """Update phase of the message and write to history log."""
+    mocker.patch(DATETIME_PATCH, conftest.MockDatetime)
+    msg = event_consumer.GEventMessage(pubsub_msg, raw_msg_data)
+
+    msg.update_phase('emo')
+
+    assert 'emo' == msg.phase
```

Comment:
What do you think of using a legit looking phase here? I'm thinking of the person who isn't familiar with Gordon and is trying to grok how phases work and this could serve as a clue.

Reply:
ha, I used this for other phase testing. I'd hope the prose docs and docstrings would help?

Reply:
(sorry this resp ^ was unclear) I didn't originally think it would be misleading but I had forgotten that some people read tests as docs. If changes are wanted, then I think a separate PR should be made to clean up the sprinkled bits of humor like this.

Reply:
Ah, you're right about this being the case elsewhere. I agree that this should be a separate PR, but I'd be curious to hear @descent3me and @smstone weight if this is worth pursuing.